src/test/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetFieldEnumUnitTest.java

```diff
@@ -87,18 +121,14 @@ public void testNonAlleleSpecificSingleAlt() {
                 .attribute(STRAND_BIAS_BY_SAMPLE_KEY, "2,0,6,0") // KCIBUL: why doesn't this work with: new int[]{2,0,6,0}  What is right?
                 .make();
 
-        builderA.attribute("AS_QUALapprox","|154|0")
-                .attribute("AS_VarDP","2|6|0")
-                .attribute("BaseQRankSum","-0.572")
-                .attribute("DP","8")
-                .attribute("MQRankSum","-0.572")
-                .attribute("MQ_DP","8")
-                .attribute("QUALapprox","154")
-                .attribute("RAW_GT_COUNT","0,1,0")
-                .attribute("RAW_MQ","5024.00")
-                .attribute("RAW_MQandDP","5024,8")
-                .attribute("ReadPosRankSum","1.067")
-                .attribute("VarDP","8")
+        builderA.attribute(AS_RAW_QUAL_APPROX_KEY,"|154|0")
+                .attribute(AS_VARIANT_DEPTH_KEY,"2|6|0")
+                .attribute(DEPTH_KEY,"8")
+                .attribute(MAP_QUAL_RANK_SUM_KEY,"-0.572")
+                .attribute(RAW_QUAL_APPROX_KEY,"154")
+                .attribute(RAW_MAPPING_QUALITY_WITH_DEPTH_KEY,"5024,8")
```

Comment:
Is MQ=25 what we expect?  Definitely unusual, so I'm just checking.

Reply:
I don't understand where 25 is coming from?  If it helps, these are just wiring tests, so scientifically unusual values are fine (ie we're just testing the math/transformation of those values is correct)